apps/notifications/src/panel/templates/block-user.jsx

```diff
@@ -147,28 +145,27 @@ export class UserBlock extends React.Component {
 		if ( home_url ) {
 			return (
 				<div className="wpnc__user">
-					<a className="wpnc__user__site" href={ home_url } target="_blank">
+					<a className="wpnc__user__site" href={ home_url } target="_blank" rel="noreferrer">
```

Comment:
Noting that we generally prefer ` rel="noreferrer noopener"` instead of just `noreferrer`.

Reply:
I'll bring it back to be consistent, but it is not required according to https://github.com/yannickcr/eslint-plugin-react/issues/2022. 

tldr: The eslint rule was checking for both, the spec says `noreferrer` is enough and someone did a test in many browsers and found that is the case.

Reply:
Back when I was looking at why we're adding both `noreferrer` and `noopener` I recall I didn't find much information about `noopener`, nor browser support information. I guess it's safe to say it doesn't hurt to keep it in place just in case.

Reply:
The https://github.com/yannickcr/eslint-plugin-react/issues/2022 discussion concludes that there are browsers that remove `window.opener` only on `rel="noreferrer"` and _won't_ remove it on `rel="noreferrer noopener"`, because they support `noreferrer` and don't understand the `noopener` attribute, so fail to parse the latter 🤦 

The spec (and [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Window/open#noreferrer)) says that `noreferrer` should also remove `window.opener`, and browsers either do it, or don't do it at all, even with `noopener` added.

There are no browsers that would:
- not remove `window.opener` on mere `rel="noreferrer"`
- *and* would start removing it after adding `rel="noopener"`

Therefore, `noopener` never has any useful effect and `rel="noreferrer"` is always enough.

Things were probably different as late as July 2019 last year, because Firefox didn't support `noreferrer` until v68 released that month. But today, the above describes the state of the art 🙂

Reply:
Thanks for the context! What a mess! 😅